packages/components/src/Well/_docs/Well.stories.tsx

```diff
@@ -37,17 +37,41 @@ export const Playground: Story = {
   },
 }
 
-export const Variants: Story = {
+export const Colors: Story = {
   render: args => (
     <>
-      {variantTypes.map(variant => (
-        <>
-          <Heading tag="h3" variant="heading-6">
-            {variant}
-          </Heading>
-          <Well variant={variant} {...args} />
-        </>
-      ))}
+      <Heading tag="h4" variant="heading-6" classNameOverride="mb-12">
+        `white` replaces the `default` variant
+      </Heading>
+      <Well {...args} />
+      <Heading tag="h4" variant="heading-6" classNameOverride="mb-12">
+        `gray` is an alternative to the `default` variant
+      </Heading>
```

Comment:
Something like this? Because it currently sounds like if you had defined `variant="default"` then it will turn white.

```suggestion
      <Heading tag="h4" variant="heading-6" classNameOverride="mb-12">
        `white` is the new default when no color/variant is defined
      </Heading>
      <Well {...args} />
      <Heading tag="h4" variant="heading-6" classNameOverride="mb-12">
        `gray` replaces the `default` variant
      </Heading>
```

Reply:
I suppose literally in terms of the codemod and what we will do to mitigate the changes in repos, but white is the actually replacement to the default class, which is why I'm iffy stating that in the docs.

Reply:
What about: 
```
<Heading tag="h4" variant="heading-6" classNameOverride="mb-12">
  `gray` offers accessible color contrast when needing gray Wells 
</Heading>
```
@HeartSquared

Reply:
I've included the colour update for `default` to match `gray` now, so you will be able to remove the "accessible version" part.
No issues with "former" though 😄 Ship it